### PR TITLE
Updated and new API calls

### PR DESCRIPTION
**Pull request by klinejordan**
Fixed current end point calls (octet-stream content type and missing fields bugs) and added many new API end points

### CODE REVIEW

File: Power BI API/Power BI API/PBIAPI.pq [hunk 1]
```diff
@@ -26,6 +26,11 @@ windowHeight = 1024;
 //
 // Exported function(s)
 //
+MyFx = (path as text) =>
```
Comment (migueesc123) [inline Power BI API/Power BI API/PBIAPI.pq]: Hey! this function appears to be not related to the query.

Comment (migueesc123) [inline Power BI API/Power BI API/PBIAPI.pq]: Hey! this function appears to be not related to the query.

**Comment by klinejordan** [comment]
changes made and pushed
